modules/nativoBidAdapter.js

```diff
@@ -451,6 +459,89 @@ export const spec = {
 registerBidder(spec)
 
 // Utils
+export class RequestData {
+  constructor() {
+    this.bidRequestDataSources = []
+  }
+
+  addBidRequestDataSource(bidRequestDataSource) {
+    if (!(bidRequestDataSource instanceof BidRequestDataSource)) return
+
+    this.bidRequestDataSources.push(bidRequestDataSource)
+  }
+
+  processBidRequestData(bidRequest, bidderRequest) {
+    for (let bidRequestDataSource of this.bidRequestDataSources) {
+      bidRequestDataSource.processBidRequestData(bidRequest, bidderRequest)
+    }
+  }
+
+  getRequestDataQueryString() {
+    if (this.bidRequestDataSources.length == 0) return
+
+    const queryParams = this.bidRequestDataSources.map(dataSource => dataSource.getRequestQueryString()).filter(queryString => queryString !== '')
+    return queryParams.join('&')
+  }
+}
+
+export class BidRequestDataSource {
+  constructor() {
+    this.type = 'BidRequestDataSource'
+  }
+  processBidRequestData(bidRequest, bidderRequest) { }
+  getRequestQueryString() { return '' }
+}
+
+export class UserEIDs extends BidRequestDataSource {
+  constructor() {
+    super()
+    this.type = 'UserEIDs'
+    this.qsParam = new QueryStringParam('ntv_pb_eid')
+    this.values = new Set()
+  }
+
+  processBidRequestData(bidRequest, bidderRequest) {
+    const eids = bidRequest.userIdAsEids
+
+    if (eids == undefined) return
+
+    for (let eid of eids) {
+      if (this.values.has(eid)) {
```

Comment:
I believe this will in practice always be false. It's only true if the same object (meaning the same identity & memory location) appears twice in the array, which would be a bug for which a fix would make more sense than this error notification.

Stepping back, what's the intent here? even if you meant to check for duplicates by value, that's still something that I don't see happening.

Reply:
@dgirardi The intent is to determine if eid values could differ across bidRequests. I noticed that each bidRequest has a userIdAsEids property but didn't know if they contained the same values or could ever be different. This was an initial attempt to get some insight into whether these values would ever be different but I see your point regarding the this.values.has(eid) check. 

Do you have any insight into whether userIdAsEids values can differ across bidRequests? I can adjust the logic accordingly.

Reply:
Within one invocation to `buildRequests`, all the request objects [reference the same `userIdAsEids`](https://github.com/prebid/Prebid.js/blob/30c61f8f9bcae47f10bb37faec1c364d82059488/modules/userId/index.js#L487-L494).  So I was wrong, this would be true every time you have more than one bid request (I missed that you are reusing the same set across them).

Across different invocations (different auctions) the contents of `userIdAsEids` can change. 

If you're wondering why it's `bidRequest.userIdAsEids` instead of `bidderRequest.userIdAsEids` - good question. I have no answer.

Reply:
Let me roll back a bit - by value they are identical, but now that I'm reading that code again I'm not sure they would be the same reference. But if I interpret your concern as "do I need to worry about different slots having different user ids", the answer is no.

Reply:
Thank you @dgirardi for the insight. I will adjust accordingly on my end. Please stand by

Reply:
I have removed the duplicate check